dev/ory/oathkeeper_rules.yaml

```diff
@@ -10,11 +10,25 @@
     handler: allow
   mutators:
     - handler: noop
+
+- id: allow-clear-cookies-endpoint
+  upstream:
+    url: "http://e2e-tests:4012"
+  match:
+    url: "<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/clearCookies"
+    methods: ["GET", "POST", "OPTIONS"]
+  authenticators:
+    - handler: anonymous
+  authorizer:
+    handler: allow
+  mutators:
+    - handler: noop
+
 - id: galoy-backend
   upstream:
     url: "http://e2e-tests:4012"
   match:
-    url: "<(http|https)>://<.*><[0-9]+>/graphql<.*>"
+    url: "<(http|https)>://<[a-zA-Z0-9-.:]+>/<(?!auth/clearCookies)(.*)>"
```

Comment:
@krtk6160 

I had to change this because there are a few routes that were missed.  I added the matcher to exclude the `auth/clearCookies` endpoint. if this is not set then it has two matches and won't resolve. this endpoint needs to allow a user to pass expired or revoked cookies so the server can clear them for the browser and avoid login issues. the matcher still allows all wildcard endpoints, such as `graphql`.